core/src/main/java/org/infinispan/configuration/global/GlobalTracingConfiguration.java

```diff
@@ -0,0 +1,98 @@
+package org.infinispan.configuration.global;
+
+import java.util.Objects;
+
+import org.infinispan.commons.configuration.attributes.Attribute;
+import org.infinispan.commons.configuration.attributes.AttributeDefinition;
+import org.infinispan.commons.configuration.attributes.AttributeSet;
+
+/**
+ * Configuration for metrics. See {@link GlobalTracingConfigurationBuilder}.
+ */
+public class GlobalTracingConfiguration {
+
+   public static final AttributeDefinition<String> COLLECTOR_ENDPOINT = AttributeDefinition.builder(org.infinispan.configuration.parsing.Attribute.COLLECTOR_ENDPOINT, null, String.class).immutable().build();
+   public static final AttributeDefinition<Boolean> ENABLED = AttributeDefinition.builder(org.infinispan.configuration.parsing.Attribute.ENABLED, true, Boolean.class).immutable().build();
+   public static final AttributeDefinition<TracingExporterProtocol> EXPORTER_PROTOCOL = AttributeDefinition.builder(org.infinispan.configuration.parsing.Attribute.EXPORTER_PROTOCOL, TracingExporterProtocol.OTLP).immutable().build();
+   public static final AttributeDefinition<String> SERVICE_NAME = AttributeDefinition.builder(org.infinispan.configuration.parsing.Attribute.SERVICE_NAME, "infinispan-server").immutable().build();
+
+   static AttributeSet attributeDefinitionSet() {
+      return new AttributeSet(GlobalTracingConfiguration.class, COLLECTOR_ENDPOINT, ENABLED, EXPORTER_PROTOCOL, SERVICE_NAME);
+   }
+
+   private final AttributeSet attributes;
+   private final Attribute<String> collectorEndpoint;
+   private final Attribute<Boolean> enabled;
+   private final Attribute<TracingExporterProtocol> exporterProtocol;
+   private final Attribute<String> serviceName;
```

Comment:
I took `GlobalMetricsConfiguration` as model to create this class. Actually it seems I choose the wrong model :)

Reply:
It was the model we used in the past. IMO, it is a waste of memory since those attributes are read when the cache manager or a cache is started. Why caching something that is read only once?